src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -141,6 +141,26 @@ public int MaxDepth
             }
         }
 
+        /// <summary>
+        /// Specifies the policy used to convert a property's name on an object to another format, such as camel-casing.
+        /// The resulting property name is expected to match the JSON payload during deserialization, and
+        /// will be used when writing the property name during serialization.
+        /// </summary>
+        /// <remarks>
+        /// This property can be set to <see cref="JsonPropertyNamingPolicy.CamelCase"/> to specify a camel-casing policy.
+        /// </remarks>
```

Comment:
Worth also mentioning here that using `[JsonName(...)]` supersedes this?

Reply:
Done. Thanke.